recipes/zfp/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "zfp" %}
```

Comment:
You should change the name to `zfpy` since this is the python wrapper.

Reply:
Okay